acceptance_tests/test_register_page.py

```diff
@@ -1,49 +1,46 @@
-import unittest
+from django.test import TestCase
 
 from page_objects import PageObject, PageElement
 from selenium import webdriver
 
 
-class RegisterPage(PageObject):
-    username = PageElement(name="username")
-    password1 = PageElement(name="password1")
-    password2 = PageElement(name="password2")
-    register = PageElement(class_name="submit-register")
+class RegistrationPage(PageObject):
+    username = PageElement(name='username')
+    password1 = PageElement(name='password1')
+    password2 = PageElement(name='password2')
+    registration_button = PageElement(class_name='submit-register')
 
 
-class RegisterPageValidation(unittest.TestCase):
+class RegisterPageValidation(TestCase):
 
     def setUp(self):
         self.driver = webdriver.Chrome()
-        self.driver.get("http://127.0.0.1:8080/users/register/")
-        self.page = RegisterPage(self.driver)
-
-    def test_invalid_username_entry(self):
-        self.page.username = "test"
-        self.page.password1 = "pass1234"
-        self.page.password2 = "pass1234"
-        self.page.register.click()
-        error_message = self.driver.find_element_by_id("username-error").text
-        self.assertEqual(error_message, "A user with that username already exists.")
-
-    def test_invalid_password_entry(self):
-        self.page.username = "new_user"
-        self.page.password1 = "pass123"
-        self.page.password2 = "pass123"
-        self.page.register.click()
-        error_message = self.driver.find_element_by_id("password2-error").text
-        self.assertEqual(error_message, "This password is too short. It must contain at least 8 characters.")
-
-    def test_different_password_entry(self):
-        self.page.username = "new_user"
-        self.page.password1 = "pass1234"
-        self.page.password2 = "pass12345"
-        self.page.register.click()
-        error_message = self.driver.find_element_by_id("password2-error").text
-        self.assertEqual(error_message, "The two password fields didn't match.")
+        self.driver.get('http://127.0.0.1:8080/users/register/')
```

Comment:
you should DRY the host data somewhere. Short term solution can be setUp method of the test case class, but long term solution should be a test settings file or a common parent class for all acceptance tests.

Once you have a variable for host data (in this case: 'http://127.0.0.1:8080/'), you can use Python's urllib to build the URL you need using common host and a relative page URL.